docs/EN_US/ECLWatch/ECLWa_mods/SessFaq.xml

```diff
@@ -0,0 +1,250 @@
+        inactivity.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Does Auto Refresh of
+        active workunits and graphs extend your ESP session?</para>
+
+        <para><emphasis role="strong">A</emphasis>: No, only user actions such
+        as typing or mouse clicks extend a session.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Will I have to login to
+        ECLWatch?</para>
+
+        <para><emphasis role="strong">A</emphasis>: Yes, just as you currently
+        do.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Will I have to login to
+        the ECL IDE?</para>
+
+        <para><emphasis role="strong">A</emphasis>: Yes, but you already
+        should be. No perceptible changes here only behind the scenes where
+        you are being authenticated.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Will I have to login to
+        ConfigManager?</para>
+
+        <para><emphasis role="strong">A</emphasis>: No.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: What credentials should I
+        use to login with?</para>
+
+        <para><emphasis role="strong">A</emphasis>: Use your assigned
+        credentials.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Can I log out of ECL
+        Watch?</para>
+
+        <para><emphasis role="strong">A</emphasis>: Yes, there is a link to
+        logout available. You are able to log off, and if you do not your
+        account locks after a configurable period of inactivity.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Will my sessions get
+        logged off due to inactivity?</para>
+
+        <para><emphasis role="strong">A</emphasis>: No. After a configurable
+        period of inactivity your session locks. You then need to unlock to
+        resume your session.</para>
+      </listitem>
+
+      <listitem>
+        <?dbfo keep-together="always"?>
+
+        <para><emphasis role="strong">Q</emphasis>: How long until my password
+        expires?</para>
+
+        <para><emphasis role="strong">A</emphasis>: This depends on your
+        system policies and the configured security manager.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Can I log in concurrently
+        with different credentials?</para>
+
+        <para><emphasis role="strong">A</emphasis>: Yes, using different tabs
+        in a single browser, multiple instances of the same browser, or
+        multiple instances of different browsers.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Will I be able to log in
+        as a different user?</para>
+
```

Comment:
Duplicate question

Reply:
Not really. This is very similar, but not the same.

Reply:
Now that you have changed the order, it works